src/layout/Divider/config.ts

```diff
@@ -0,0 +1,25 @@
+import { CG } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const DIVIDER_SUMMARY_OVERRIDE_PROPS = new CG.obj()
+  .extends(CG.common('ISummaryOverridesCommon'))
+  .optional()
+  .setTitle('Summary properties')
+  .setDescription('Properties for how to display the summary of the component')
+  .exportAs('DividerSummaryOverrideProps');
+
+export const Config = new CG.component({
+  category: CompCategory.Action,
```

Comment:
I would probably call this a presentation component. It's certainly not possible for the user to perform an action on it..

Reply:
Great catch! I agree 😁